airflow/settings.py

```diff
@@ -165,6 +167,15 @@ def configure_orm(disable_connection_pool=False):
 except:
     pass
 
+# Import custom airflow logging file `airflow_logging.py` from AIRFLOW_HOME,
```

Comment:
Any reason we can't reuse the existing "airflow_local_settings.py" logic in settings.py? We could basically copy what is done for the policy function in that file. That way we can centralize the logic for loading custom modules.